library/Icingadb/Widget/ItemList/HistoryListItem.php

```diff
@@ -231,10 +231,20 @@ protected function assembleTitle(BaseHtmlElement $title)
 
                 break;
             case 'state_change':
-                if ($this->item->state->state_type === 'hard') {
-                    $title->add(t('Hard state changed'));
+                $state = $this->item->state === 'hard' ?
+                    $this->item->state->hard_state : $this->item->state->soft_state;
```

Comment:
If you need to break a line with ternary operators (? :) please break each result with its respective operator.

```suggestion
                $state = $this->item->state === 'hard'
                    ? $this->item->state->hard_state
                    : $this->item->state->soft_state;
```